scripts/quests/otherAreas/Elderly_Pursuits.lua

```diff
@@ -0,0 +1,157 @@
+-----------------------------------
+-- Elderly Pursuits
+-----------------------------------
+-- Log ID: 4, Quest ID: 75
+-----------------------------------
+--Pre-requisite Completed Secrets of Oven Lost Log ID: 4, Quest ID: 73
+-----------------------------------
+-- Despachiaire !gotoid 16883721
+-- Rouva !gotoid 17719492
+--- Para NM ??? !gotoid 16785771
+-----------------------------------
+local ID = zones[xi.zone.CARPENTERS_LANDING]
+-----------------------------------
+
+local quest = Quest:new(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.ELDERLY_PURSUITS)
+
+quest.reward =
+{
+    item     = xi.item.ELEGANT_RIBBON,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE and
+            player:hasCompletedQuest(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.SECRETS_OF_OVENS_LOST)
+        end,
+
+        [xi.zone.TAVNAZIAN_SAFEHOLD] =
+        {
+            ['Despachiaire'] = quest:progressEvent(517),
+
+            onEventFinish =
+            {
+                [517] = function(player, csid, option, npc)
+                    quest:begin(player)
+                    npcUtil.giveKeyItem(player, xi.ki.ANTIQUE_AMULET)
+                end,
+            },
+        },
+    },
+
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_ACCEPTED and vars.Prog <= 2
+        end,
+
+        [xi.zone.SOUTHERN_SAN_DORIA] =
+        {
+            ['Rouva'] =
+            {
+                onTrigger = function(player, npc)
+                    if
+                        player:hasKeyItem(xi.ki.ANTIQUE_AMULET)
+                    then
+                        return quest:progressEvent(747)
+                    end
+                end,
+            },
+
+            onEventFinish =
+            {
+                [747] = function(player, csid, option, npc)
+                    quest:setVar(player, 'Prog',1)
+                end,
+            },
+        },
+
+        [xi.zone.CARPENTERS_LANDING] =
+        {
+            ['qm_para'] =
+            {
+                onTrigger = function(player, npc)
+                    if
+                        quest:getVar(player, 'Prog') == 1 and
+                        npcUtil.popFromQM(player, npc, ID.mob.PARA, { claim = true, hide = 0 })
+                    then
+                        return quest:messageSpecial(ID.text.STENCH_OF_DECAY)
+                    elseif
+                        quest:getVar(player, 'Prog') == 2 and
+                        player:hasKeyItem(xi.ki.ANTIQUE_AMULET)
+                    then
+                        npcUtil.giveKeyItem(player, xi.ki.CATHEDRAL_MEDALLION)
+                        quest:setVar(player, 'prog', 3)
```

Comment:
These vars are case-sentitive. This would never set the value of `Prog` to 3, what happened when you were testing this?

Reply:
From testing it does not appear to be case sensitive in this case since the value is set to 3. Will update to make sure things are consistent.

Reply:
These strings are piped directly down into the database, and then looked up exactly as they are:
```cpp
    auto FetchCharVar(uint32 charId, std::string const& varName) -> std::pair<int32, uint32>
    {
        const char* fmtQuery = "SELECT value, expiry FROM char_vars WHERE charid = %u AND varname = '%s' LIMIT 1";
```

So they're very much case-sensitive. Also, looking at other quests it looks like the convention is the capitalised `'Prog'`